kube/patch.sh

```diff
@@ -30,6 +30,20 @@ spec:
         - \"--stackdriver.kubernetes.cluster-name=${KUBE_CLUSTER}\"
         #- \"--stackdriver.generic.location=${GCP_REGION}\"
         #- \"--stackdriver.generic.namespace=${KUBE_CLUSTER}\"
+        #- \"--web.enable-statusz\"
+        env:
+          - name: NODE_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: spec.nodeName
+          - name: POD_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
+          - name: POD_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
```

Comment:
This will go straight into our [Getting Started docs](https://cloud.google.com/monitoring/kubernetes-engine/prometheus#configuration) and I'm concerned about clutter. What do you think about adding this to the [updating the configuration section](https://cloud.google.com/monitoring/kubernetes-engine/prometheus#updating_the_configuration) instead?

Reply:
How about creating a copy the file for docs? Alternatively you could ask one of our tech writers to give a thumbs up on the larger snippet. I'd do it outside this PR, so you can submit the rest as-is.

Reply:
Ok, removed these for now.